src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -370,6 +371,13 @@ def load_arguments(self, _):
         c.argument('container_name', container_name_type, required=True, help='Name of the Cosmos DB sql container name')
         c.argument('location', options_list=['--location', '-l'], help="Location of the account", required=True)
 
+    # Retrive MongoDB Collection Backup Info
+    with self.argument_context('cosmosdb mongodb retrieve-latest-backup-time') as c:
```

Comment:
```suggestion
    with self.argument_context('cosmosdb mongodb collection retrieve-latest-backup-time') as c:
```
It should be per collection isnt it?